openwisp-config/files/openwisp.agent

```diff
@@ -121,6 +121,14 @@ while [ -n "$1" ]; do
 			export POST_REGISTRATION_HOOK="$2"
 			shift
 			;;
+		--checksum_max_retries)
```

Comment:
for consistency with the parameters above, this should be `checksum-max- retries`